lib/puppet/defaults.rb

```diff
@@ -831,6 +831,13 @@ def self.default_vendormoduledir
       :type    => :string,
       :desc    => "A lock file to indicate that the ssl bootstrap process is currently in progress.",
     },
+    :ssl_lockfile_timeout => {
+      :default => "5s",
```

Comment:
should it just be "5" without the "s" ?

Reply:
It does look like a typo, but since this is a `:ttl` type setting, it takes an optional unit like `5s`, `10m`, `1h`, etc